src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -26,7 +26,7 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.UUID;
-// used for using FetchOptions 
+// used for using FetchOptions
```

Comment:
the import seems self explanatory...you can remove this comment here.

Reply:
This and the function getTotalMessageCount come from a change made by @AnnaKawakami , they're here because I rebased master in this branch. Should I still make these changes?

Reply:
You can delete the self-explanatory comment! Also, did you mean this this the new master branch? I can remove the function if so

Reply:
No, this is not a new master branch but I added things from master for it to not have any conflicts! What do you think? @gauravmishra